server/sandbox_run.go

```diff
@@ -301,7 +302,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	logPath := filepath.Join(logDir, id+".log")
 
 	// Handle https://issues.k8s.io/44043
-	if err := ensureSaneLogPath(logPath); err != nil {
+	if err := libkpod.EnsureSaneLogPath(logPath); err != nil {
```

Comment:
Used in one other place in `server` right now (git grep says RunPodSandbox). Once we get that moved over into libkpod, we can make it private again.